source/benchmark.rst

```diff
@@ -0,0 +1,104 @@
+Benchmarking your Pulsar setup
+==============================
+
+To determine the performance of your Pulsar setup, you can use the
+`GalaxyBenchmarker <https://github.com/AndreasSko/Galaxy-Benchmarker>`_.
```

Comment:
Would it be ok for you if we move this to usegalaxy-eu?

Reply:
Sure 👍